DOCUMENTATION.md

```diff
@@ -0,0 +1,127 @@
+First include the package in your file like this:
+```python
+from ttnmqtt import MQTTClient as mqtt
+```
+
+### MQTTClient
+
+The class constructor can be called following this scheme:
+```python
+mqtt(appID, appAccessKey, [mqttAddress])
+```
+- `appID`: **string**  this the name you gave your application when you created it.
+- `appAccessKey`: **string**  it can be found at the bottom of your application page under **ACCESS KEYS**.
+- `mqttAddress`: **string**  this the address of the handler to which your application was registered. If you registered to a personnal or private handler, please provide the address, if not you don't need to provide this argument. Please make sure you also give the port number when providing the mqttAddress.
+All the above informations can be found in your The Things Network console.
+The constructor returns an **MQTTClient object** set up with your application informations, ready for connection.
+
+### connect
+This function connect your client, in case you closed it and which to open the connection once again.
+```python
+client.connect()
+```
+
+### close
+Disconnects the MQTT client from which we call the method. Also able to stop a forever loop in case the client was running on a loop launched by the `startForever()` method.
+```python
+client.close()
+```
+
+### startForever
+A background loop is started by default when creating an MQTT client but in in case you wish to start a loop as the main loop of your process. You can start a forever loop with this method. You wont be able to run anything else at the same time on this script.
+```python
+client.startForever()
+```
+
+### start
+Starts a loop for the client in the background so that it's possible to run another process (such as a web server) in the same script. This loop is started by default when creating the MQTT client. So it should only be used in case the client was stopped and you wish to start it again after re-connecting.
+```python
+client.start()
+```
+
+### stop
+Stops the MQTT client and also disconnect it.
+```python
+client.stop()
+```
+
+### Using Callbacks
+
+The callback functions are functions which are executed when a trigger event happens. They should be set right after the MQTT client creation so that they don't miss any event.
+
+#### setUplinkCallback
+Set the callback function, to be called when an uplink message is received.
+```python
+client.setUplinkCallback(uplinkCallback)
+```
+
+##### uplinkCallback
+The callback function must be declared in your script following this structure:
+* `uplinkCallback(msg, client)`
+  * `msg`: **JSON object**  the message received by the client
+  * `client`: **object**  the client from which the callback is executed are calling
+
+On each message reception, you should see **receving message from** in the console, and the callback will be executed.
```

Comment:
We should remove that - developers who build applications and scripts on-top of our MQTT library should be able to have a clean output.